src/Data/String/CodeUnits.purs

```diff
@@ -297,9 +297,9 @@ dropWhile :: (Char -> Boolean) -> String -> String
 dropWhile p s = drop (countPrefix p s) s
 
 -- | Returns the substring at indices `[begin, end)`.
--- | If either index is negative, it is normalised to `length s - index`,
--- | where `s` is the input string. `""` is returned if either
--- | index is out of bounds or if `begin > end` after normalisation.
+-- | If either index is negative, it is normalised to `length s + index`,
+-- | where `s` is the input string. `""` is returned if
+-- | `begin > end` after normalisation.
```

Comment:
I would recommend backing out this unrelated change, just to keep the PR more straightforward and avoid bikeshedding this comment.

Reply:
Yep sorry for this one.